### PR TITLE
Adds post request on RB review

### PR DESCRIPTION
**Pull request by itsjoekent**
#### What's this PR do?

Sends a post request to Vision when a RB is reviewed
#### How should this be reviewed?

Does it send the request successfully?
Is the payload ok?
#### Any background context you want to provide?

🏃 ::message-broker: 
(thats me running away)
#### Relevant tickets

Fixes #SNACK_WEEK
#### Checklist
- [ ] Test on staging but leave it off afterwards

### CODE REVIEW

File: lib/modules/dosomething/dosomething_reportback/includes/ReportbackItem.php [hunk 1]
```diff
@@ -198,6 +198,17 @@ public function review($values) {
       $this->deleteFile();
     }
 
+    // Send external message request
+    $vision_url = variable_get('dosomething_settings_vision_url');
```
Comment (angaither) [inline lib/modules/dosomething/dosomething_reportback/includes/ReportbackItem.php:202]: this should probably be in it's own function

**Comment by angaither** [comment]
this is fine, but let's not merge stuff from snack week.
if you wanna continue branching off of this branch and make PRs against that, that would be 👍

**Comment by itsjoekent** [comment]
sounds like branch mania

**Comment by angaither** [comment]
global 2.0

**Pull request closed without merging**